docs/openshift/integration.md

```diff
@@ -0,0 +1,17 @@
+# OpenShift Cluster on Azure
+
+OpenShift orchestrator configuration in acs-engine is based on Kubernetes configuration as OpenShift is a Kubernetes distribution with additional features.
+
+OpenShift cluster deployed with acs-engine contains following components: router, registry, service catalog and template service broker.
+
+## Differences between OpenShift and Kubernetes in acs-engine
+
+Besides masters and nodes, Openshift introduces additional node group for running infrastructure services, like router and internal registry. Router is responsible for distributing ingress traffic to correct services running in OpenShift cluster. Highly available OpenShit clusters contain at least 3 infrastructure nodes, where every node is running one router. To evenly distribute HTTP and HTTPS traffic to all routers, `Microsoft.Network/loadBalancers` and `Microsoft.Network/publicIPAddresses` resources are utilized in Azure and `Microsoft.Network/networkSecurityGroups` resource allows this kind of traffic.
```

Comment:
HA Openshift cluster may be also interpreted as a HA master setup. Isn't there any better wording for HA router?